test/integration/addon/assertion_test.go

```diff
@@ -14,9 +14,9 @@ import (
 	addonapiv1alpha1 "open-cluster-management.io/api/addon/v1alpha1"
 )
 
-const addOnDefaultConfigSpecHash = "287d774850847584cc3ebd8b72e2ad3ef8ac6c31803a59324943a7f94054b08a"
-const addOnTest1ConfigSpecHash = "d76dad0a6448910652950163cc4324e4616ab5143046555c5ad5b003a622ab8d"
-const addOnTest2ConfigSpecHash = "3f815fe02492288fd235ed9bd881987aebb6f15fd2fa2b37c982525c293679bd"
+const addOnDefaultConfigSpecHash = "e374ecc7f9ab844fd9ccdf4630a8bd317a8e1821716b215f219b4d3b10f12680"
```

Comment:
please add a TODO here. I think we need a better way to handle this hardcoded hash, it will break everytime api is changed.

Reply:
Added.